ftva_lab_data/tests.py

```diff
@@ -1,3 +1,112 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from django.contrib.auth.models import User, Group, Permission
+from django.urls import reverse
+from ftva_lab_data.models import SheetImport
 
-# Create your tests here.
+
+class UserAccessTestCase(TestCase):
+    """Tests expected behavior for different users requesting various views."""
+
+    fixtures = ["sample_data.json"]
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        # Create client and SheetImport object for tests
+        cls.client = Client()
+        cls.test_object = SheetImport.objects.get(pk=2)
+        cls.test_form_data = {"file_name": "test_file_name.txt"}
+
+        # Create two users, one to add to editors group and one with no group
+        cls.unauthorized_user = User.objects.create_user(
+            username="unauthorized", password="testpassword"
+        )
+        cls.authorized_user = User.objects.create_user(
+            username="authorized", password="testpassword"
+        )
+
+        # Create editors group and add permissions to add and edit SheetImport
```

Comment:
Since the editors group is now defined in a fixture, load the fixture and use that definition instead of creating the group here in the tests.  That is different from how we did it in the OH Staff UI project tests... but those tests just checked group membership, rather than actual permissions as you do.  Your overall approach to permissions is better than we did in OH, so the tests can be better too :)